File: Lista fabio (soluções)/ESTRUTURA REPETIÇÃO 04 WHILE/Atividade L/questao07.py
'''
Leia um número N, some todos os números inteiros entre 1 e N e escreva o resultado obtido.
'''

def main():
    numero = int(input("numero: "))
    soma = 0
    c = 1

    while c <= numero:
        soma += c
        c+=1
    print(f"Soma dos número de 1 e {numero} = {soma}")

main()

File: Lista fabio (soluções)/ESTRUTURA REPETIÇÃO FOR/Atividade P/questao07.js
// Leia um número N, some todos os números inteiros entre 1 e N e escreva o resultado obtido.

import { get_number } from "./utils/io_utils.js"

function main(){
    let n = get_number("N value: ")
    let soma = 0
    console.log("valores: ")
    for (let i = 1; i <= n; i++){
        console.log(`${i}`)
        soma += i
    }
    console.log(`Soma: ${soma}`)
}

main()